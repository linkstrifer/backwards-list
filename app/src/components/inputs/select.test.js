import React from 'react';
import renderer from 'react-test-renderer';

import SelectComponent from './select.component';

test('Select snapshot', () => {
  const component = renderer.create(
    <SelectComponent />
  );

  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
