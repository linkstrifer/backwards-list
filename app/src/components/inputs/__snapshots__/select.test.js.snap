// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select snapshot 1`] = `
<div>
  <label
    htmlFor="input-order"
    style={
      Object {
        "marginRight": "10px",
      }
    }
  >
    Ordenar:
  </label>
  <select
    id="input-order"
    onChange={[Function]}
  >
    <option
      defaultValue={true}
      value="asc"
    >
      a...z
    </option>
    <option
      value="des"
    >
      z...a
    </option>
  </select>
</div>
`;
